src/renderer/helpers/utils.js

```diff
@@ -53,33 +53,36 @@ export function toLocalePublicationString ({ publishText, isLive = false, isUpco
   } else if (isRSS) {
     return publishText
   }
-  const strings = publishText.split(' ')
-  // filters out the streamed x hours ago and removes the streamed in order to keep the rest of the code working
-  if (strings[0].toLowerCase() === 'streamed') {
-    strings.shift()
-  }
-  const singular = (strings[0] === '1')
+
+  const match = publishText.match(PUBLISHED_TEXT_REGEX)
+  const singular = (match[1] === '1')
   let translationKey = ''
-  switch (strings[1].substring(0, 2)) {
+  switch (match[2].substring(0, 2)) {
```

Comment:
Does substring work if the second number is greater than the length? 
Also wanted to make sure that months still work?

Reply:
YouTube uses `mo` as the shortening for months, as `m` is already used for minutes.

As for your first question, I'm not quite sure what you are asking, is it what happens if the regex fails?

Reply:
Oh I think I understood your first question now, yes substring works if the string is shorter than 2 characters.
To quote mdn:
> Any argument value that is less than 0 or greater than str.length is treated as if it were 0 and str.length, respectively.

So a string with just one character is passed in, e.g. `d`, javascript uses str.length, so 1, instead of the 2 specified in the second argument.